seqr/views/apis/anvil_workspace_api.py

```diff
@@ -125,6 +126,17 @@ def grant_workspace_access(request, namespace, name):
 
     return create_json_response({'success': True})
 
+
+@anvil_workspace_access_required(meta_fields=['workspace.bucketName'])
+def get_anvil_vcf_list(request, namespace, name, workspace_meta):
+    bucket_name = workspace_meta['workspace']['bucketName']
+    bucket_path = 'gs://{bucket}'.format(bucket=bucket_name.rstrip('/'))
+    data_path_list = [path.replace(bucket_path, '', 1) for path in get_gs_file_list(bucket_path)
```

Comment:
is there a reason you are adding the `count` parameter in the `replace` call here? In general its best to just use default behavior for optional parameters unless there is a strong reason not to, as it keeps things more readable

Reply:
I use `1` for the `count` parameter to avoid accidentally replacing more than one occurrence of the pattern. But maybe you are right. It doesn't look like that `gs://` could have more occurrences in a path. I can take out the `count`.